src/gobupload/__init__.py

```diff
@@ -31,38 +31,43 @@ def get_report(contents, events, recompares):
     return counted_events
```

Comment:
PR is OK but rebase onto develop gives conflicts. Please rebase and push branch. Thanks